Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -61,27 +60,64 @@ public bool CanScroll
         }
 
         /// <summary>
-        /// Edit modes for defining the clipping box masking boundaries.
+        /// Edit modes for defining scroll viewable area and scroll interaction boundaries.
         /// </summary>
         public enum EditMode
         {
             Auto = 0, // Use pagination values
-            Manual, // Use direct manipulation of the clipping box object
+            Manual, // Use direct manipulation of the object
         }
 
         [SerializeField]
         [Tooltip("Edit modes for defining the clipping box masking boundaries. Choose 'Auto' to automatically use pagination values. Choose 'Manual' for enabling direct manipulation of the clipping box object.")]
         private EditMode maskEditMode;
 
         /// <summary>
-        /// Edit modes for defining the clipping box masking boundaries. Choose 'Auto' for automatically use pagination values. Choose 'Manual' for enabling direct manipulation of the clipping box object.
+        /// Edit modes for defining the clipping box masking boundaries. Choose 'Auto' to automatically use pagination values. Choose 'Manual' for enabling direct manipulation of the clipping box object.
         /// </summary>
         public EditMode MaskEditMode
         {
             get { return maskEditMode; }
             set { maskEditMode = value; }
         }
 
+        [SerializeField]
+        [Tooltip("Edit modes for defining the scroll interaction collider boundaries. Choose 'Auto' to automatically use pagination values. Choose 'Manual' for enabling direct manipulation of the collider.")]
+        private EditMode colliderEditMode;
+
+        /// <summary>
+        /// Edit modes for defining the scroll interaction collider boundaries. Choose 'Auto' to automatically use pagination values. Choose 'Manual' for enabling direct manipulation of the collider.
+        /// </summary>
+        public EditMode ColliderEditMode
+        {
+            get { return colliderEditMode; }
+            set { colliderEditMode = value; }
+        }
+
+        [SerializeField]
+        [HideInInspector]
+        private bool maskEnabled = true;
+
+        /// <summary>
+        /// Visibility mode of scroll content. Default value will mask all objects outside of the scroll viewable area.
+        /// </summary>
+        public bool MaskEnabled
+        {
+            get { return maskEnabled; }
+            set 
+            {
+                if (!value && value != wasMaskEnabled)
+                {
+                    RestoreContentVisibility();
+                }
+                wasMaskEnabled = value;
+                maskEnabled = value;
+            }
+        }
+
+        // Workaround for serializing mask enabled as the editor dot not handle property setters
```

Comment:
```suggestion
        // Workaround for serializing mask enabled as the editor does not handle property setters
```

Reply:
Could you actually add a little more on what the workaround is? Or maybe it's actually clear to everyone else and not me, in which case I can follow up offline with people who know what's up